src/report/report-generator.ts

```
@@ -1,8 +1,4 @@
-import {
-  EvaluationReport,
-  EvaluationConfig,
-  EvaluationResult,
-} from '../types/evaluation';
+import { EvaluationConfig, EvaluationReport, EvaluationResult } from './evaluation';
```

ESLint

[error] Replace `·EvaluationConfig,·EvaluationReport,·EvaluationResult·` with `⏎··EvaluationConfig,⏎··EvaluationReport,⏎··EvaluationResult,⏎`

(prettier/prettier)